### PR TITLE
Rename top folders of vDVS

### PR DESCRIPTION
**Pull request by luomiao**
vmdk_plugin -> client_plugin
plugin -> plugin_dockerbuild

This is the first PR for Issue: https://github.com/vmware/docker-volume-vsphere/issues/1551
This PR is only for renaming purpose.
Refactoring of common utility functions will be another separate PR.

### CODE REVIEW

**Comment by shaominchen** [comment]
From the comments in #1551, one suggestion is to rename "plugin" to "plugin_config". That makes sense to me. Can we get finalized on the decision before making the change?

**Comment by luomiao** [comment]
@shaominchen 
In my opinion I prefer to name it to plugin_dockerbuild :)
It does include config.json but Dockerfile is equally important inside...
So as Mark has suggested, it's the real managed plugin is built here.
So I think it's better to still use managed_plugin (or plugin_dockerbuild :) )

**Comment by luomiao** [comment]
@msterin agreed and updated :)

**Comment by luomiao** [comment]
Update:
vmdk_plugin -> client_plugin
plugin -> plugin_dockerbuild

File: Commonvars.mk [hunk 1]
```diff
@@ -61,4 +61,4 @@ MANAGED_PLUGIN_NAME := "vsphere:latest"
 
 # build places binaries here:
 BIN := ../build
-MANAGED_PLUGIN_LOC := ../plugin
\ No newline at end of file
+MANAGED_PLUGIN_LOC := ../plugin_dockerbuild
```
Comment (shuklanirdesh82) [inline Commonvars.mk]: https://github.com/vmware/docker-volume-vsphere/blob/master/misc/scripts/deploy-tools.sh

files needs to be changed too.

Comment (luomiao) [inline Commonvars.mk]: Done. Can you please approve?

**Pull request merged**